bin/deploy

```diff
@@ -121,71 +118,37 @@ function getEnvStatus(env) {
         production: process.env.PROD_URL
     };
 
-    return rp({
-        url: URLs[env] + '/status'
-    }).then(response => {
-        return JSON.parse(response);
+    return request({
+        url: URLs[env] + '/status',
+        json: true
     });
 }
 
-function getCommitsToBeDeployed() {
-    let lookups = [getEnvStatus('test'), getEnvStatus('production')];
-
-    // get server statuses
-    return Promise.all(lookups).then(responses => {
-        let test = responses[0];
-        let prod = responses[1];
-
-        // get the commit IDs
-        let ids = {
-            test: test.COMMIT_ID,
-            prod: prod.COMMIT_ID || 'c54660f391ed5542308d0deb4b87f07ce54ec120' // last build before adding commit_id
```

Comment:
I've removed this on the fairly safe assumption that we'll never deploy this commit as it's over two years old